react-components/src/desktop/view/UserMenu.js

```diff
@@ -0,0 +1,120 @@
+                break;
+            case build(ids.DESKTOP, ids.COLLABORATORS_LINK):
+                presenter.onCollaboratorsClick();
+                break;
+
+            case build(ids.DESKTOP, ids.USER_MANUAL_LINK):
+                presenter.onDocumentationClick();
+                break;
+
+            case build(ids.DESKTOP, ids.INTRO_LINK):
+                this.doIntro();
+                break;
+            case build(ids.DESKTOP, ids.ABOUT_LINK):
+                presenter.onAboutClick();
+                break;
+
+            case build(ids.DESKTOP, ids.LOGOUT_LINK):
+                presenter.doLogout(false);
+                break;
+            default:
+                break; //do nothing
+        }
+    }
+
+    render() {
+        const {anchorEl} = this.state;
+        const classes = this.props.classes;
+        return (
+            <span>
+                    <img className={classes.menuIcon}
+                         src={userImg}
+                         alt="User Menu"
+                         onClick={this.onUserMenuClick}
+                         ref={this.userBtn}>
+                    </img>
+                <Menu id={build(ids.DESKTOP, ids.USER_PREF_MENU)} anchorEl={anchorEl}
+                      open={Boolean(anchorEl)}
+                      onClose={this.handleClose}>
+                    <MenuItem id={build(ids.DESKTOP, ids.PREFERENCES_LINK)}
+                              onClick={this.onMenuItemSelect}>
+                        <DEHyperlink text={getMessage("preferences")}/>
+                    </MenuItem>
+                    <MenuItem id={build(ids.DESKTOP, ids.COLLABORATORS_LINK)}
+                              onClick={this.onMenuItemSelect}>
+                        <DEHyperlink text={getMessage("collaboration")}/>
+                    </MenuItem>
+                      <Divider />
+                    <MenuItem id={build(ids.DESKTOP, ids.USER_MANUAL_LINK)}
+                              onClick={this.onMenuItemSelect}>
+                        <DEHyperlink text={getMessage("documentation")}/>
+                    </MenuItem>
+                    <MenuItem id={build(ids.DESKTOP, ids.INTRO_LINK)}
+                              onClick={this.props.doIntro}>
+                        <DEHyperlink text={getMessage("introduction")}/>
+                    </MenuItem>
+                    <MenuItem id={build(ids.DESKTOP, ids.ABOUT_LINK)}
+                              onClick={this.onMenuItemSelect}>
+                        <DEHyperlink text={getMessage("about")}/>
+                    </MenuItem>
+                     <Divider />
+                    <MenuItem id={build(ids.DESKTOP, ids.LOGOUT_LINK)}
+                              onClick={this.onMenuItemSelect}>
+                        <DEHyperlink text={getMessage("logout")}/>
+                    </MenuItem>
+                </Menu>
+            </span>
+        );
+    }
+}
+
+export default injectSheet(styles)(withI18N(UserMenu, intlData));
```

Comment:
Should we use material-ui `withStyles` instead of `injectSheet`?

Reply:
Yes. We can get rid of `injectSheet`. (and remove jss dep from our package.json i think)